### ISSUE TITLE
Previewed file changed when QTTabBar is used

### ISSUE THREAD

**Issue by Geeyun-JY3** [xupefei/QuickLook#726]
### **TLDR: turn off QTTabBar -> Options -> Folder View -> Enhanced incremental search in view**

----------------

**Describe the bug**
Support [QTTabBar](http://qttabbar.wikidot.com/qttabbar)

**To Reproduce**
While QTTabBar installed, unable to peek any file or folder selected. Instead, it will peek another file in the folder (most of time it will peek particular one file if there are not too many files).
***
（防止上方蹩脚英语看不懂也用中文写一次）
在安装有 QTTabBar 的情况下，在任意文件夹中选中文件后，按空格，有时候会预览该文件夹中特定的某个文件；如果该文件夹中文件较多，则预览结果虽不一定都是同一个文件，但都是某几个文件中的一个。

**Expected behavior**
QuickLook and QTTabBar work normally and are compatible with each other.

**Desktop (please complete the following information):**
 - OS Version: Windows 10 1909
 - QuickLook Version: 3.6.9 from the `msi` installer
